pkgs/development/compilers/tinycc/default.nix

```
@@ -1,50 +1,31 @@
 { lib
 , stdenv
-, fetchFromRepoOrCz
-, copyPkgconfigItems
-, makePkgconfigItem
+, fetchFromGitHub
 , perl
 , texinfo
 , which
 }:
 
 stdenv.mkDerivation rec {
   pname = "tcc";
-  version = "unstable-2022-07-15";
+  version = "0.9.27+date=2022-05-26";
```

```suggestion
  version = "0.9.27";
```
The normal version format has no dates. We only do that for the unstable one.